src/Altinn.Apps/KubernetesWrapper/deployment/values.yaml

```diff
@@ -0,0 +1,33 @@
+replicaCount: 1
+
+image:
+  # Set "repository" name of your image for manual Helm install and upgrade.
+  repository: tddregistry.azurecr.io/kuberneteswrapper
+  tag: latest
+  pullPolicy: Always
+  pullSecrets:
+    # Change this to the name of your pull secret
+    - name: acr-secret
+  serviceAccount: kuberneteswrappersa
+
+service:
+  name: kuberneteswrapper
+  type: ClusterIP
+  externalPort: 80
+  # If your application is running on another port, change only the internal port.
+  internalPort: 80
+
+ingress:
+  enabled: true
+  annotations:
+    traefik.ingress.kubernetes.io/rewrite-target: /
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
+  paths:
+    # The first path will automatically be overwritten during deploy
+    # Additional paths needs to be configured manually
+    - path: /kuberneteswrapper
```

Comment:
will this work? Isnt it /[controller] where controller is deployment as the code is written now?

Reply:
Yes, works like a charm. Will give the path .../kuberneteswrapper/deployments

Reply:
We should follow the documentation agreed upon when it comes to api urls 
so the url should api/vi/deployments ...

Also there look like there are some comments that should be removed/changed in this file

Reply:
Fixed.

Reply:
Still believe that the path should be /api/v1/deployments for this to work, as we have done with register, profile and storage

Reply:
Yes, that is the path for the controller. This path just tells us where the app itself is located. I assume you do not want to have all applications pointing to the same path? Then we would have to ensure unique controller names for each app? Sounds weird?
Just to be clear. in this case the path would be .../kuberneteswrapper/api/v1/deployments

Reply:
You can test it out live if you want ;) http://tdd.apps.at21.altinn.cloud/kuberneteswrapper/deployments?labelSelector=release=tdd-helloworld

(Have not updated the deployments after the controller route was changed from deployments to /api/v1/deployments, but you get the idea)

Reply:
That is how its done in platform

Reply:
@hannekot I suggest we keep it as it is until we have agreed upon the way API paths should be structured.

Reply:
Will set up a meeting to discuss this